partners.yml

```diff
@@ -0,0 +1,546 @@
+      summary: Send the Tink code fetched with the Tink link.
+      requestBody:
+        $ref: "#/components/requestBodies/sendCode"
+      responses:
+        200:
+          $ref: "#/components/responses/empty"
+        400:
+          $ref: "#/components/responses/dataError"
+        401:
+          $ref: "#/components/responses/authError"
+        500:
+          $ref: "#/components/responses/serverError"
+        503:
+          $ref: "#/components/responses/serviceError"
+      security:
+      - basic_auth: []
+      
+  /application:
+    post:
+      tags:
+      - Signup endpoints
+      summary: Send a new application to Moank
+      requestBody:
+        $ref: "#/components/requestBodies/create"
+      responses:
+        200:
+          $ref: "#/components/responses/extendedApplication"
+        400:
+          $ref: "#/components/responses/dataError"
+        401:
+          $ref: "#/components/responses/authError"
+        500:
+          $ref: "#/components/responses/serverError"
+        503:
+          $ref: "#/components/responses/serviceError"
+      security:
+      - basic_auth: []
+
+  /application/:applicationId/event:
+    post:
+      tags:
+      - Signup endpoints
+      summary: Send event about an application
+      requestBody:
+        $ref: '#/components/requestBodies/event'
+      responses:
+        200:
+          $ref: "#/components/responses/empty"
+        400:
+          $ref: "#/components/responses/dataError"
+        401:
+          $ref: "#/components/responses/authError"
+        500:
+          $ref: "#/components/responses/serverError"
+        503:
+          $ref: "#/components/responses/serviceError"
+      security:
+      - basic_auth: []
+
+  /application/:applicationId:
+    get:
+      tags:
+      - Application endpoints
+      summary: Fetch an application
+      parameters:
+      - name: applicationId
+        in: path
+        description: Application's id as set in the application response object
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          $ref: "#/components/responses/extendedApplication"
+        400:
+          $ref: "#/components/responses/dataError"
+        401:
+          $ref: "#/components/responses/authError"
+        500:
+          $ref: "#/components/responses/serverError"
+      security:
+      - basic_auth: []
+
+components:
+  requestBodies:
+    getLink:
+      required: true
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+            - socialSecurityNumber
+            properties:
+              socialSecurityNumber:
+                type: string
+                format: YYYYMMDDXXXX
+                description: Swedish personnummer in format YYYYMMDDXXXX
+                example: "197202104332"
+              bank:
+                type: string
+                description: Customer's bank name
+                example: "SEB"
```

Comment:
Here I think we need to have a list of supported banks and only accept values in the enum. We might need to require that field because I don't think we want customers to be able to connect any bank in Tink but only one of the supported ones.

Reply:
I agree here. We could define a general type and reference it here like with the other one.